src/main/java/com/epam/restbookservice/services/BookService.java

```diff
@@ -0,0 +1,42 @@
+package com.epam.restbookservice.services;
+
+import com.epam.restbookservice.domain.Book;
+import com.epam.restbookservice.repositories.BookRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Service;
+import org.springframework.web.server.ResponseStatusException;
+
+import static java.lang.String.format;
+
+@Service
+@RequiredArgsConstructor
+public class BookService {
+
+    private final BookRepository bookRepository;
+
+    public Book saveBook(Book book) {
+        return bookRepository.save(book);
+    }
+
+    public Iterable<Book> getBooks() {
```

Comment:
You can use JpaRepository in Bookrepository to return List<Book> instead of Iterable